src/bdc/steps/regionalatlas.py

```diff
@@ -90,25 +92,23 @@ def run(self) -> DataFrame:
             return self.empty_result
```

Comment:
Maybe we should add a check for merged_geo.json to the verify() method instead of returning an empty df here as this will probably override all previous data right?

Reply:
I changed it to the verify() method in the style like we have for the gpt_summarizer. A StepError is thrown if the merged.geojson does not exist